app/src/androidTest/java/com/duckduckgo/cookies/impl/features/firstparty/RealFirstPartyCookiesModifierTest.kt

```diff
@@ -255,7 +256,8 @@ class RealFirstPartyCookiesModifierTest {
         val time = Instant.now()
             .plus(expiryTimeInSeconds, ChronoUnit.SECONDS)
             .atOffset(ZoneOffset.UTC)
-            .format(DateTimeFormatter.ofPattern("EEE, d MMMM yyyy HH:mm:ss Z"))
+            .format(DateTimeFormatter.ofPattern("EEE, d MMMM yyyy HH:mm:ss Z").withLocale(Locale.UK))
```

Comment:
Nit: `Locale.US` might be better default since we have that across the codebase already

Reply:
Done!  👍